sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -74,7 +74,7 @@ def get_refresh_token(self, service, **kwargs):
     def exchange_aad_token_for_refresh_token(self, service=None, **kwargs):
         # type: (str, Dict[str, Any]) -> str
         refresh_token = self._client.authentication.exchange_aad_access_token_for_acr_refresh_token(
-            service=service, access_token=self._credential.get_token(self.credential_scope).token, **kwargs
+            service=service, access_token=self._credential.get_token(*self.credential_scopes).token, **kwargs
```

Comment:
I don't know Python very well ... what does the `*` before `self.credential_scopes` do?

Reply:
It's a variadic argument.